core/src/epicli/data/common/ansible/playbooks/roles/vault/files/configure-vault.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+# Description: This script perform configuration of Hashicorp Vault to be used with Epiphany
+# More information you can find in Epiphany documentation in HOWTO.md
+# TODO: Revoke root token
+# TODO: Policy for non-root access
+
+HELP_MESSAGE="Usage: configure-vault.sh -c SCRIPT_CONFIGURATION_FILE_PATH -a VAULT_IP_ADDRESS"
+
+function print_help { echo "$HELP_MESSAGE"; }
+
+function log_and_print {
+    local string_to_log="$1";
+    echo "$string_to_log" | tee -a configure-vault.log;
+}
+
+function exit_with_error {
+    local string_to_log="$1";
+    log_and_print "$string_to_log";
+    exit 1;
+}
+
+function check_vault_error {
+    local exit_code ="$1";
```

Comment:
Inconsistent variable naming, please consider converting it to upper case letters.

Reply:
Retracted due to "naming convetions".